certified-connectors/timeghost/apiProperties.json

```diff
@@ -84,7 +84,7 @@
         }

```

Comment:
@ytognder Please let us know why this file has added, Can you please make sure that you have merged your branch with `dev `branch to make a PR to `dev `branch